src/main/java/com/monumental/models/Monument.java

```diff
@@ -52,23 +60,23 @@
     @Column(name = "description")
     private String description;
 
-    @Column(name = "inscription")
+    @Column(name = "inscription", length = 1024)
```

Comment:
what does this do? make the max length 1024 chars? if so that's probably too short

Reply:
That's just what I found to work for the initial data load.

I'll increase it to 2048.